fedora_elections/templates/base.html

```diff
@@ -90,6 +90,20 @@ <h4 class="hide">Main Site Links:</h4>
     </div>
 
     {% block jscripts %}
+      <script type="text/javascript" src="{{
+        url_for('static', filename='jquery-1.10.2.min.js') }}"> </script>
+      {% if config['FEDMENU_URL'] %}
+      <script src="{{ config['FEDMENU_URL'] }}/js/fedora-libravatar.js"></script>
```

Comment:
Oh, shoot.  You don't actually need ``fedora-libravatar.js`` **unless** you're specific users in the call to ``fedmenu(...)``.

This line can be removed (although it doesn't harm anything..).